### ISSUE TITLE
problem viewing the different images of 4Q207Anonymous User Submission

### ISSUE THREAD

**Issue by Bronson-Brown-deVost** [Scripta-Qumranica-Electronica/ScrollEditor#59]
Submitter: Jonathan Ben Dov
I have a problem viewing the different images of 4Q207. They seem to stack on each other in an imperfect way.

**Comment by Bronson-Brown-deVost** [comment]
I need to tackle this with a quick analysis of the image stack.  Sometimes the images are rotated 180°, so I will just start by searching for that.

The information about the relative positioning of images is stored in the database and is sent in the ImageStackDTO under the `transformToMaster` property. I need to change this to: scale, rotate, translateX, translateY (as in our other positional DTOs).

The web UI must parse that information and transform the image when it is placed in the stack.

To sum up, there are three tasks:
1. Analyse images in stack and detect the difference in orientation
2. Update the transform of the image stack in the database to be discreet scale, rotate, translate instead of a transform matrix (update the API to reflect this change)
3. Update the web UI to read the transform values in the ImageStackDTO and apply them to the images when they are displayed.

**Comment by tsviar** [comment]
I dont see the problem in 4Q207. there are 2 artefacts, as shown in the snapshot.
![image](https://user-images.githubusercontent.com/16132723/115994576-5018ec00-a5e0-11eb-9324-2bdcf2247215.png)

**Comment by Bronson-Brown-deVost** [comment]
@tsviar, this is a problem with the stack of image in the artefact and imaged object editor (see the picture below).  Ingo and I spoke with Itay about this issue last Monday.  I have made the necessary changes to the database and the API (I also performed the image analysis to determine if any images in the stack are upside down).

We will wait just a moment before fixing this on the front end, because the IAA may be able to fix the rotation problem on their end.

In the long term, though, the front end will need to read this positional data (rotation, scale, translation) for lining up the older PAM images and also any newer images that are produced in the future.

![Screen Shot 2021-04-25 at 16 28 13](https://user-images.githubusercontent.com/2952608/115997623-0cc07c80-a5e4-11eb-8ee5-54727b7f48e4.png)

**Comment by Bronson-Brown-deVost** [comment]
Orit is rotating these images on their iiif server, so we will not need to do anything to fix this in the web app.  Once the upload to the iiif server is complete, the issue will be fixed without any further intervention from us.
